configs/olcf/job_script.olcf.bash

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# comment out if using debug queue
+#PBS -q batch
+# comment in to get the debug queue (only available on Titan)
+##PBS -q debug
+# change number of nodes to change the number of parallel tasks
+# (anything between 1 and the total number of tasks to run)
+#PBS -l nodes=10
+#PBS -l walltime=1:00:00
+#PBS -A cli115
+#PBS -N mpas_analysis
+#PBS -o mpas_analysis.o$PBS_JOBID
+#PBS -e mpas_analysis.e$PBS_JOBID
+
+cd $PBS_O_WORKDIR   # optional, since this is the default behavior
+
+export OMP_NUM_THREADS=1
+
+module unload python python/base
```

Comment:
I'm guessing that `python/base` may be a module specific to NERSC.  It doesn't hurt to unload something that doesn't exist but it could be confusing.  But I could also be wrong and this could be needed on OLCF, too.